### PR TITLE
Updated AboutUs

### PR DESCRIPTION
**Pull request by tzihiang**
Changed name of pic

### CODE REVIEW

**Review by teo-jun-xiong — APPROVED** [review]

**Review by hans555 — APPROVED** [review]

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -37,11 +37,11 @@ Responsibilities: Data
 '''
 
 === Tay Zi Hiang Willis
-image::m133225.jpg[width="150", align="left"]
-{empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
+image::tzihiang.jpg[width="150", align="left"]
```
Comment (gohkh) [inline docs/AboutUs.adoc]: Sorry I didn't notice this earlier, the website says that the file should be named as ".png" even if it's in jpg format.

Comment (teo-jun-xiong) [reply docs/AboutUs.adoc]: i think reuploading a png will be good. i tried with a jpg and it didn't work

Comment (tzihiang) [reply docs/AboutUs.adoc]: ok! i'll edit and push a new branch for that

Comment (teo-jun-xiong) [reply docs/AboutUs.adoc]: you can edit in your existing branch. after you are done editing, commit and then push it. the changes will reflect here

**Review by shanecsj — APPROVED** [review]

**Review by gohkh — APPROVED** [review]

**Pull request merged**